android/jni/app-android.cpp

```diff
@@ -1011,9 +1012,12 @@ extern "C" bool JNICALL Java_org_ppsspp_ppsspp_NativeActivity_runEGLRenderLoop(J
 		ELOG("Error: Surface is null.");
 		return false;
 	}
-	
+
+retry:
+
 	bool vulkan = g_Config.iGPUBackend == GPU_BACKEND_VULKAN;
 
+	int tries = 0;
```

Comment:
I don't think it is, but you're right, I should probably rewrite it the obvious way instead.